modules/core/src/main/java/org/apache/ignite/internal/processors/rest/client/message/GridClientAuthenticationRequest.java

```diff
@@ -50,13 +51,23 @@ public void credentials(Object cred) {
     @Override public void writeExternal(ObjectOutput out) throws IOException {
         super.writeExternal(out);
 
+        U.writeString(out, login());
+        U.writeString(out, password());
+
+        U.writeMap(out, userAttrs);
```

Comment:
Should be serialized after `cred` (perhaps, in this case, compatibility will be supported)

Reply:
fixed